components/inputs/test/input-checkbox.vdiff.js

```diff
@@ -148,14 +148,28 @@ describe('d2l-input-checkbox', () => {
 			template: html`<d2l-input-checkbox aria-label="Label for checkbox"></d2l-input-checkbox>`
 		},
 		{
-			name: 'spacer',
+			name: 'supporting',
```

Comment:
I pre-seeded these vdiff goldens with copies of the existing "spacer" ones. I'm not expecting any diffs -- they should be identical.